### ISSUE TITLE
Custom card view drawRect never fires

### ISSUE THREAD

**Issue by santanu4ver** [Yalantis/Koloda#14]
Hi,

Instead of returning UIImageView to DraggableCardView *contentView*, I've added a custom UIView and this seems working fine. However I've added drawRect() in my custom UIView Class, and set background color as clearColor in XIB:

```
override func drawRect(rect: CGRect) {
        // shaping for rounded corner
        var path = UIBezierPath(roundedRect: rect, byRoundingCorners: UIRectCorner.AllCorners, cornerRadii: CGSize(width: 6.0, height: 6.0))
        UIColor.whiteColor().setFill()
        path.fill()
    }
```

I saw this works fine for first 3 cards (white background color with rounded shape), but any cards after first 3 count is coming with clear background, I notice drawRect() never fires for those views. I've tried call the method someway, but failed.

Can you suggest a way out? 

Thanks!

**Comment by AEugene** [comment]
Hi,
Issue is fixed. Update your Koloda.

**Comment by AEugene** [comment]
Thank you. Your issue fixed by #15

**Comment by santanu4ver** [comment]
Thank you, for the updates! This seems working fine now!
